pagopa/io-app
# feat: [IOPAE-1311] Bottom sheet displayed when the user taps on the settings icon in the services section

## Short description
This PR introduces a bottom sheet displayed when the user taps on the settings icon in the services section.
The bottom sheet allows the user to navigate to services preferences or the main settings.

<details open><summary>Details</summary>
<p>

| services  |
| - |
| <video src="https://github.com/user-attachments/assets/84eac23c-ff1d-420f-9c0d-090b5ae6a594" width="300"/> |

</p>
</details> 

## List of changes proposed in this pull request
- added `useMessageContactsBottomSheet` hook
- updated `HeaderFirstLevelHandler` to display the bottom sheet if `isSettingsVisibleAndHideProfile` is true. 
- updated locales

## How to test
Using `io-dev-api-server`, navigate to the services tab and tap the settings icon. Check that the bottom sheet is visible and the navigation works as expected.
